src/components/DropZone/utils/index.ts

```diff
@@ -1,4 +1,4 @@
-import {DropZoneEvent} from '../types';
+type DropZoneEvent = DragEvent | React.ChangeEvent<HTMLInputElement>;
```

Comment:
This type was only used in this file, so inline it instead of having a separate types.ts file.